pystac/utils.py

```diff
@@ -605,3 +605,8 @@ def is_file_path(href: str) -> bool:
     """
     parsed = urlparse(href)
     return bool(os.path.splitext(parsed.path)[1])
+
+
+def is_url(href: str) -> bool:
```

Comment:
Can we add a quick docstring here since it's in our public API now?

Reply:
eh on second read it's only really is_url if your only other option is local, so I made it private again.